### ISSUE TITLE
Suggest more structure to Intent section of 3.3.7 Understanding article

### ISSUE THREAD

**Issue by ChrisXKeroack** [w3c/wcag#2650]
The Intent section of [Understanding Success Criterion 3.3.7: Accessible Authentication](https://w3c.github.io/wcag/understanding/accessible-authentication.html) is long and covers multiple supporting points and topics. But those points and topics don't seem to align with the criteria and topic of 3.3.7. 

It would greatly help understanding of the Intent section if it were organized with subheadings that map to the "provides at least one of the following" items as listed in the success criterion definition.  In other words, since 3.3.7 calls out that a cognitive function test has to provide an **alternative**, a completion **mechanism** for that test, a test of **object recognition** or identifying **personal content**, the Intent portion of the article should be organized into subheadings of **Alternative**, **Mechanism**, **Object Recognition** and **Personal Content**. 

However, the Intent section seems to discuss background of 2-factor auth, autofill, copy/paste, CAPTCHA and show/hide of password fields, so either there needs to be...

a) more content specific to explaining alternatives, cognitive function test mechanisms, and examples of valid cognitive function tests (object recognition or personal content verification), or 
b) revision of the text of 3.3.7 to more closely match what's discussed in the Intent section.

**Comment by alastc** [comment]
Hi @ChrisXKeroack, there has been a fair amount of work on the criterion text recently, and the understanding document does need an overhaul now.

**Comment by hidde** [comment]
Not sure if it warrants a separate issue, but I was particulartly confused by: 

> If a [CAPTCHA](https://www.w3.org/TR/turingtest/) is used as part of an authentication process, there must be a method that does not include a cognitive function test.

in the Understanding doc, this “must” seems to say the opposite of the “Object Recognition” clause in the criterion text.

**Comment by bruce-usab** [comment]
Good catch @hidde.  I don't think 3.3.7 is quite up to date with the current phrasing of the SC.  Also, I think we try to avoid `must` in the Informative documents.

Here is an attempt at replacing that sentence:

> If a CAPTCHA is used as part of an authentication process, this SC requires that the CAPTCHA be limited to (presumably simple) object recognition, or that an authentication option be provided which avoids the CAPTCHA (or any other cognitive function test).

**Comment by alastc** [comment]
This was discussed in the backlog call, the to-do is for Jemma: Overhaul the intent section to line up with the new SC structure. And integrate Bruce's point above about CAPTCHA.
